shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/ValueInternal.hs

```diff
@@ -121,106 +125,7 @@ instance Era era => Val (Value era) where
       addrHashLen :: Integer
       addrHashLen = 28
 
--- =====================================================================================
```

Comment:
Given the `CanonicalMap` stuff has moved, can we now just move the core value definition into `Value` and drop the internal module?

Reply:
Good idea. It is done.